style.css

```diff
@@ -112,3 +112,235 @@ body {
     width: 30%;
   }
 }
+
+.work {
+  background-color: #e5e5e5;
+  padding: 6rem 1.5rem 1.5rem 1.5rem;
+  display: grid;
+  grid-template-rows: 1fr;
+  grid-template-columns: 1fr;
+}
+
+.recentWorks {
+  display: flex;
+  flex-direction: column;
+  justify-content: space-between;
+  align-items: center;
+  height: 6.688rem;
+  width: 100%;
+  margin-bottom: 3rem;
+}
+
+.recentWorks > h2 {
+  font-family: "Crete Round", serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 2.5rem;
+  line-height: 3.25rem;
+}
+
+.projects {
+  display: flex;
+  flex-direction: column;
+  justify-content: space-between;
+  gap: 1rem;
+  width: 100%;
+  padding: 1rem 1.5rem 1.5rem 1.5rem;
+}
+
+.projects > h2 {
+  font-family: "Crete Round", serif;
+  white-space: nowrap;
+  font-style: normal;
+  color: #091e42;
+  font-weight: 400;
+  font-size: 3rem;
+  line-height: 2.75rem;
+}
+
+.projects > p {
+  color: #091e42;
+}
+
+.tools {
+  list-style: none;
+  display: flex;
+  justify-content: space-between;
+}
+
+.toolsBtn {
+  color: #091e42;
+  border-color: #8993a4;
+  padding: 0.625rem 0.75rem;
+}
+
+.mainSeeBtn {
+  padding: 0.75rem;
+  color: #fff;
+  font-family: "Inter", serif;
+  font-style: normal;
+  font-weight: 700;
+  font-size: 1.063rem;
+  line-height: 1.5rem;
+  text-align: center;
+  letter-spacing: 0.03em;
+  max-width: fit-content;
+  height: 3rem;
+  background: #ff6b00;
+}
+
+.cardContainer {
+  margin-top: 2.5rem;
+  margin-bottom: 4rem;
+}
+
+.card {
+  background-image: url("./images/card-bg.png");
+  background-repeat: no-repeat;
+  background-size: cover;
+  height: 24.125rem;
+  display: flex;
+  flex-direction: column;
+  gap: 0.75rem;
+  padding: 0 1rem;
+}
+
+.card h2 {
+  font-family: "Crete Round", serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 2rem;
+  line-height: 2.75rem;
+  color: #fff;
+  margin-top: 6.25rem;
+}
+
+.cardSeeProj {
```

Comment:
- [ ] Just as the previous reviewer mentioned, Kindly add the right button interaction styles. I noticed that you added the hover styles but it is still missing some features that need to be corrected so please fix this :+1: .
- [ ] I also noticed that you didn't add the `pressed ` styles so kindly add those as well. I have attached a video below for your reference. :wink: 
- Video Reference
 
[Screencast from 02-16-2023 08:05:45 PM.webm](https://user-images.githubusercontent.com/102699958/219475189-b6081e53-f050-4d57-96f2-a634644f4c76.webm)

- [ ] From the video, you can also see that you still have the scroll effect that the previous reviewer mentioned and this might be because of one or more elements overflowing their containers so kindly find out what the source could be and solve that as well.